Wrappers/Python/cil/optimisation/algorithms/SIRT.py

```diff
@@ -77,21 +76,22 @@ class SIRT(Algorithm):
 
     The preconditioning arrays (weights) :code:`M` and :code:`D` used in SIRT are defined as
 
-    .. math:: M = \frac{1}{A*\mathbb{1}} = \frac{1}{\sum_{j}a_{i,j}}
+    .. math:: M = \frac{1}{A\mathbb{1}} = \frac{1}{\sum_{j}a_{i,j}}
 
-    .. math:: D = \frac{1}{A*\mathbb{1}} = \frac{1}{\sum_{i}a_{i,j}}
+    .. math:: D = \frac{1}{A^T\mathbb{1}} = \frac{1}{\sum_{i}a_{i,j}}
```

Comment:
Should this be 
`.. math:: D = \frac{1}{A^T\mathbb{1}} = \frac{1}{\sum_{i}a_{j,i}}`
 or am I mis-understanding?

Reply:
I think it was correct but looking closer there is no "i" on the other side of the equation so was wrong in another way. I just deleted it because I think it was confusing,